### PR TITLE
Add noracooks.com

### PR DESCRIPTION
**Pull request by brianstrauch**

### CODE REVIEW

**Comment by jknndy** [comment]
Thanks for the contribution! 
Two small changes needed before this is good to go. The `__init__.py` entry for `from .noracooks import NoraCooks` belongs down on line 283 right above norecipes, this is a known bug. Also it looks like this site provides coverage for a few more fields that can be included in the json (category, cook_time, prep_time, cuisine, ratings, ratings_count, nutrients & keywords). There is no need to add them to the scraper class but coverage can be added to the test file. Thanks again!

**Comment by brianstrauch** [comment]
Thanks for the review, updated! @jknndy

File: tests/test_data/noracooks.com/noracooks.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "fatContent": "25 g",
+    "saturatedFatContent": "4 g",
+    "carbohydrateContent": "71 g",
+    "sugarContent": "53 g",
+    "proteinContent": "4 g",
+    "sodiumContent": "408 mg",
+    "fiberContent": "4 g"
+  },
+  "image": "https://www.noracooks.com/wp-content/uploads/2018/07/IMG_8885.jpg",
+  "keywords": ["vegan chocolate cake"]
```
Comment (jknndy) [inline tests/test_data/noracooks.com/noracooks.json:63]:
```suggestion
  "keywords": [
    "vegan chocolate cake"
  ]
```

Comment (jknndy) [reply tests/test_data/noracooks.com/noracooks.json:63]: Small formatting change here

**Comment by jknndy** [comment]
Looks good to me, thanks again for the contribution!

**Pull request merged**